src/components/App.jsx

```diff
@@ -1,18 +1,28 @@
 import React from 'react';
 
 import Header from './Header/Header.jsx';
+import SmallImages from './SmallImages.jsx';
 import LargeImage from './LargeImage.jsx';
 
 const App = () => {
   let styles = {
     padding: '0',
     margin: '0'
   }
+  let sampleImages = [
```

Comment:
are you changing this later to the data you get from db?

Reply:
yes. everything is static right now